lambdas/delete-user-data/tests/index.test.ts

```diff
@@ -12,7 +11,7 @@ jest.mock("../src/utils/send-audit-event", () => ({
 }));
 
 describe("handler", () => {
-  const mockContext = ContextExamples.helloworldContext;
+  const mockContext = {} as Context;
```

Comment:
We could create a proper mock, but it's not actually used for anything (it's passed into the logger, but that's mocked in tests)